code/game/machinery/computer/buildandrepair.dm

```diff
@@ -385,8 +385,7 @@
 							return
 						playsound(get_turf(src), 'sound/items/Welder.ogg', 50, 1)
 						user.visible_message("[user] welds the frame back into metal.", "You weld the frame back into metal.", "You hear welding.")
-						var/obj/item/stack/sheet/metal/M = getFromPool(/obj/item/stack/sheet/metal, src.loc)
-						M.amount = 5
+						new /obj/item/stack/sheet/metal/(src.loc, 5)
```

Comment:
This fixes the bug in the description but doesn't merge the two stacks, which may not be a problem at all, but if there's a way to do it, might as well do it. Does anyone know?